Intersect.Server/Localization/Strings.cs

```diff
@@ -497,6 +497,12 @@ public sealed class CraftingNamespace : LocaleNamespace
             [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
             public readonly LocalizedString nospace = @"You do not have enough inventory space to craft {00}!";
 
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+            public readonly LocalizedString CraftFailure = @"The attempt to create the item {00} failed!";
+
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+            public readonly LocalizedString LostItems = @"You lost the materials!";
```

Comment:
Could you give me some idea? as to why it won't always be that the player will lose the items, so I thought it might be necessary to let you know when you lose them. I don't like the idea of two lines either, but I couldn't think of anything else at that time.

Now I could only think about this:
![image](https://user-images.githubusercontent.com/63019821/163982693-5ee0430f-21ff-4c36-9911-6b9d30d006bf.png)
![image](https://user-images.githubusercontent.com/63019821/163982751-6a27b68e-653c-461c-b4c4-a0aa24297bdb.png)